includes/views/advanced-tools.php

```diff
@@ -57,6 +57,33 @@
 			</form>
 		</div>
 		<!-- /clear_tmp -->
+		<!-- clear_semaphore_expired_locks -->
+		<div class="tool">
```

Comment:
It's not only related to this PR, but maybe showing a spinner icon after clicking on the tool buttons would be good. This way, customers can be certain that the button is functioning and that a process is underway. 
What do you think?

Reply:
I think we could do that in a new issue because we could add to other tools as well. Do you want to do that?

Reply:
Sure.

https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/issues/843